.gitignore

```diff
@@ -5,4 +5,5 @@ terraform.tfstate*
 .terraform
 .minio.sys
 tmp
-docker/conf/certificates/*.crt
\ No newline at end of file
+docker/conf/certificates/*.crt
+2020*
```

Comment:
What is `2020`?

Reply:
It was supposed to handle the timestamp log that get's generated, but maybe it isn't required?

Reply:
Maybe use `*.log` instead?

Reply:
Nevermind, I removed it as the log file isn't there anymore

Reply:
Hm, what made it put up in the first place then? Sounds like it'd be bound to pop up again?